pkgs/by-name/gt/gtrash/package.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
+
+buildGoModule rec {
+  pname = "gtrash";
+  version = "0.0.5";
+
+  src = fetchFromGitHub {
+    owner = "umlx5h";
+    repo = pname;
```

nit: the package name and the repo name is not necessarily connected.

```suggestion
    repo = "gtrash";
```

---

thanks, I've fixed.

https://github.com/NixOS/nixpkgs/pull/281705/commits/a7c9ce94d532adea327643a6dd0b7971143e9a5c#diff-07c46b2a7822d0fafe9e1cbe26c6b2832889751d6daac49cff9c449b3b817555R13